R/pkg/NAMESPACE

```diff
@@ -1,5 +1,9 @@
 # Imports from base R
-importFrom(methods, setGeneric, setMethod, setOldClass)
+# Do not include stats:: "rpois", "runif" - causes error at runtime
+importFrom("methods", "setGeneric", "setMethod", "setOldClass")
+importFrom("methods", "is", "new", "signature", "show")
```

Comment:
Hmm, I'm not sure why but what I see is a much longer list: https://gist.github.com/felixcheung/5f16a7f489cd64eb526f7e5de3fdf2f5